Cargo.toml

```
@@ -39,6 +39,9 @@ render = ["bevy_pbr", "bevy_render", "bevy_sprite", "bevy_text", "bevy_ui"]
 png = ["bevy_render/png"]
 hdr = ["bevy_render/hdr"]
 
+# Enable the dynamic systems and components API ( useful for developing scripting solutions )
+dynamic-api = ["bevy_ecs/dynamic-api", "bevy_scene/dynamic-api"]
```

```suggestion
dynamic_api = ["bevy_ecs/dynamic_api", "bevy_scene/dynamic_api"]
```
Should we change `-` to `_` for consistency with other feature names in the main `bevy` crate?

---

Definitely. I hadn't checked the convention of the other flags. I'll update that.